src/views/html-tags/Figure.jsx

```
@@ -0,0 +1,133 @@
+                <span key={index} className={handleTextColor(word.type)}>{word.name} </span>
+              ))}
+            </code>
+          </pre>
+          
+        </figure>
+      </Card>
+      <Card>
+        <h2 className="text-xl">Figure - blockquote</h2>
+        <figure className="rounded-lg shadow-lg p-4 w-fit">
+          <blockquote className="text-gray-500 italic">
+            "If you only read the books that everyone else is reading, you can only think what everyone else is thinking."
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)